app/src/main/java/org/yuttadhammo/BodhiTimer/Models/AlarmTaskManager.kt

```diff
@@ -464,8 +464,6 @@ class AlarmTaskManager(private val mApp: Application) : AndroidViewModel(mApp) {
 
         // Update labels
         if (alarms.empty()) {
-            stopDND()
-            stopService()
```

Comment:
Why?

Reply:
stopAlarmsAndTicker() will call stopTicker(), and stopTicker() will call  stopDND() and stopService()